### PR TITLE
timelapse-deflicker: init at 142acd1

### PR DESCRIPTION
**Pull request by valeriangalliat**
Add a package for [timelapse-deflicker](https://github.com/cyberang3l/timelapse-deflicker).

Tested on macOS and NixOS.

The `ClassMethodMaker` Perl library is not a dependency of timelapse-deflicker but it's needed by `TermProgressBar`. But even though it's in its `propagatedBuildInputs`, I had an error running the program on how `ClassMethod` was not found, and adding `ClassMethodMaker` to the direct dependencies fixed it. Any idea what I'm doing wrong?

### CODE REVIEW

File: pkgs/applications/graphics/timelapse-deflicker/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    wrapProgram $out/bin/timelapse-deflicker --set PERL5LIB $PERL5LIB
+  '';
+
+  buildInputs = with perlPackages; [ makeWrapper perl PerlMagick TermProgressBar ImageExifTool FileType ClassMethodMaker ];
+
+  meta = with stdenv.lib; {
+    description = "Simple script to deflicker images taken for timelapses";
+    homepage = https://github.com/cyberang3l/timelapse-deflicker;
+    license = licenses.gpl3;
+    platforms = platforms.unix;
```
Comment (Mic92) [inline pkgs/applications/graphics/timelapse-deflicker/default.nix]: Do you want to maintain this package?

Comment (valeriangalliat) [inline pkgs/applications/graphics/timelapse-deflicker/default.nix]: Yes :)

What's the syntax for that? `maintainers = [ "Valérian Galliat <val@codejam.info>" ];`? Or should I add myself to `lib/maintainers.nix`?

Comment (Mic92) [inline pkgs/applications/graphics/timelapse-deflicker/default.nix]: add yourself to maintainers.nix

**Comment by valeriangalliat** [comment]
Thanks for the nitpicks commit @Mic92, didn't know about most of that stuff, it's so much cleaner like that!

File: pkgs/applications/graphics/timelapse-deflicker/default.nix (hunk 1 above)
Comment (valeriangalliat) [inline pkgs/applications/graphics/timelapse-deflicker/default.nix]: Cool, update the PR. I assume the maintainer key must match my username on GitHub?

**Pull request merged**